test/unit/index.test.js

```diff
@@ -0,0 +1,44 @@
+const { expect } = require('chai');
+const { mapWebPacUrlToSCCURL, BASE_SCC_URL } = require('../../index.js');
+
+describe('mapWebPacUrlToSCCURL', () => {
```

Comment:
This is great. When you get to it, I'd suggest putting a bunch of narrowly missing URLs that result in 404s, to kinda define the boundaries. But this is a great start.

Reply:
Also you _could_ test which regexes are being matched, but may not be necessary